src/main/java/com/patchworkmc/annotation/AnnotationProcessor.java

```diff
@@ -12,11 +12,20 @@
 
 public class AnnotationProcessor extends ClassVisitor {
 	private Consumer<String> consumer;
-
-	public AnnotationProcessor(ClassVisitor parent, Consumer<String> consumer) {
+	private AnnotationStorage annotationStorage;
+	private String className;
+
+	public AnnotationProcessor(
+			ClassVisitor parent,
+			Consumer<String> consumer,
+			AnnotationStorage annotationStorage,
+			String className
```

Comment:
When ASM calls `ClassVisitor#visit` it gives us the class name, so it doesn't need to be passed as a parameter. See how other transformers in patchwork-patcher use that.